### PR TITLE
refactor(scan): simplify election definition handling

### PR DESCRIPTION
**Pull request by eventualbuddha**
Splits config API into individual endpoints, `/config/election` supports GET/PATCH/DELETE. GET/PATCH support `application/octet-stream` to ensure nothing tries to parse the JSON (looking at you, express). BSD client still does validation of the election, but that could be fixed later. It only ever sends the election.json data now, not the full `ElectionDefinition`.

### CODE REVIEW

File: apps/bmd/package.json [hunk 1]
```diff
@@ -96,7 +96,6 @@
     "fetch-mock": "^9.5.0",
     "history": "^4.10.1",
     "http-proxy-middleware": "^1.0.0",
-    "js-sha256": "^0.9.0",
```
Comment (benadida) [inline apps/bmd/package.json:99]: don't we need SHA256 in BMD to compute and show the election hash, since there's no backend?

**Review by benadida — APPROVED** [review]
check the sha256 in BMD requirement, and please make sure this works with a manual test through loading a ballot package, but otherwise looks good!

File: apps/bmd/package.json (hunk 1 above)
Comment (eventualbuddha) [reply apps/bmd/package.json:99]: We delegate that to `safeParseElectionDefinition` from `@votingworks/types`, which means BMD doesn't _directly_ depend on this package, so it's safe to remove it.

**Comment by eventualbuddha** [comment]
I loaded the election through an admin card using the mock module-smartcards, which is what I assume you meant, and it worked fine. Merging!

**Pull request merged**